lib/src/flutter_deck_slide.dart

```diff
@@ -331,7 +331,10 @@ class _SlideBody extends StatelessWidget {
       child: context.flutterDeck.configuration.showProgress
           ? Stack(
               alignment: Alignment.bottomCenter,
-              children: [child, const FlutterDeckProgressIndicator()],
+              children: [
+                child,
+                context.flutterDeck.globalConfiguration.progressIndicator,
```

Comment:
As discussed in another thread:
```suggestion
                context.flutterDeck.configuration.progressIndicator,
```